pkg/statsd/statsd.go

```diff
@@ -21,30 +21,31 @@ import (
 // Server encapsulates all of the parameters necessary for starting up
 // the statsd server. These can either be set via command line or directly.
 type Server struct {
-	Backends            []gostatsd.Backend
-	CloudProvider       gostatsd.CloudProvider
-	Limiter             *rate.Limiter
-	InternalTags        gostatsd.Tags
-	InternalNamespace   string
-	DefaultTags         gostatsd.Tags
-	ExpiryInterval      time.Duration
-	FlushInterval       time.Duration
-	MaxReaders          int
-	MaxParsers          int
-	MaxWorkers          int
-	MaxQueueSize        int
-	MaxConcurrentEvents int
-	MaxEventQueueSize   int
-	EstimatedTags       int
-	MetricsAddr         string
-	Namespace           string
-	PercentThreshold    []float64
-	IgnoreHost          bool
-	ConnPerReader       bool
-	HeartbeatEnabled    bool
-	HeartbeatTags       gostatsd.Tags
-	ReceiveBatchSize    int
-	DisabledSubTypes    gostatsd.TimerSubtypes
+	Backends               []gostatsd.Backend
+	CloudProvider          gostatsd.CloudProvider
+	DisableInternalMetrics bool
```

Comment:
Should be able to drop it between `ConnPerReader` and `HeartbeatEnabled` to make the linter happy.

Reply:
I usually put all the bools at the very end to ensure there are no holes in memory layout and it is more compact.